acceptance_tests/features/steps/display_trading_as_external.py

```diff
@@ -12,10 +12,10 @@
 from controllers.party_controller import add_survey, get_party_by_email
 
 
-@given('a company has a separate trading name (s01)')
-def company_has_separate_trading_name_s01(_):
-    _add_survey_for_ru_to_respondent_suppress_exception('example@example.com', '49900000006',
-                                                        _get_last_QBS_collection_exercise_id())
+@given('a company with "{ru_ref}" has a separate trading name')
```

Comment:
Better wording would be `'a company with the reference "{ru_ref}" has a separate trading name`